hibernate-core/src/test/resources/org/hibernate/orm/test/boot/database/qualfiedTableNaming/database-object-using-catalog-placeholder.hbm.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!--
+  ~ Hibernate, Relational Persistence for Idiomatic Java
+  ~
+  ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+  ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+<hibernate-mapping xmlns="http://www.hibernate.org/xsd/hibernate-mapping"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.hibernate.org/xsd/hibernate-mapping http://www.hibernate.org/xsd/hibernate-mapping/hibernate-mapping-4.0.xsd"
+				   package="org.hibernate.test.boot.database.qualfiedTableNaming"
+				   default-access="field">
+	<database-object>
+		<create>
+			CREATE OR REPLACE FUNCTION ${catalog}.catalogPrefixedAuxObject()
+			RETURNS varchar AS
+			$BODY$
+			BEGIN
+			SELECT 'test';
+			END;
+			$BODY$
+			LANGUAGE plpgsql
```

Comment:
I assume this DDL is vendor specific, so maybe we need to specify dialect (Postgres) in testing code?

Reply:
I just need to put something here. This DDL never gets executed. I'll change the code to make this clearer.

I want to test `${schema}` replacement with every dialect, so moving this to a postgres-specific test is not the right solution.